### PR TITLE
Updated the UG for Tabview and RadioButton.

### PR DESCRIPTION
**Pull request by KarthikaRavichandranSF4617**
## Description ##

- Updated the UG for Tabview and RadioButton.

### CODE REVIEW

File: MAUI/TabView/Tab-Item-Customization.md [hunk 1]
```diff
@@ -482,19 +482,23 @@ tabView.IsScrollButtonEnabled = true;
...
+                       ScrollButtonColor="Red">
      </tabView:SfTabView>
 {% endhighlight %}
 
 {% highlight C# %}
-     tabView.ScrollButtonBackgroundColor = "Violet";
-     tabView.ScrollButtonForegroundColor = "Red";
+     StackLayout stackLayout = new StackLayout();
+    var tabView = new SfTabView();
+    tabView.ScrollButtonBackground = Colors.Violet;
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Tab-Item-Customization.md:498]: Add solid color brush for background

File: MAUI/TabView/Tab-Item-Customization.md [hunk 2]
```diff
@@ -482,19 +482,23 @@ tabView.IsScrollButtonEnabled = true;
...
+    tabView.ScrollButtonColor =Colors.Red;
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Tab-Item-Customization.md:499]: check alignment.

Comment (KarthikaRavichandranSF4617) [reply MAUI/TabView/Tab-Item-Customization.md:499]: Updated the alignment properly.

File: MAUI/TabView/Tab-Item-Customization.md (hunk 1 above)
Comment (KarthikaRavichandranSF4617) [reply MAUI/TabView/Tab-Item-Customization.md]: Added solid color brush for background.

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Pull request merged**